### DISCUSSION TITLE
AI機器人ChatGPT

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#619] [General]
我也喜歡打德州撲克，那我下次也想跟訓練過的AI PK一下
https://www.1111.com.tw/news/jobns/149382

**Comment by chia-kai-chang** [comment]
能否分享一下PK的心得 :)

**Comment by Raywei0214** [comment]
像這樣的牌類，棋藝類的AI應該蠻早以前就有了畢竟大部分都可以數據化，我反倒覺得精通這些算牌或計算的人才是真的高手。當然這也是一種可以拿來訓練人類的好工具，像我自己也有下載五子棋達人，我自認我五子棋下的很好，也有網路對戰，雖然都是一堆韓國人，目前戰績是49勝3敗。是一款還不錯的輕鬆小品，偶爾也可以與朋友相互切磋

**Comment by awehao** [comment]
想當年，高中的時候在學校打德州撲克跟橋牌，人的思考思路，每個人的思路都有特定的模式，所以打久了都能稍微猜到對方的策略，不過若是AI普及，每次跟AI打，都能產生新的思路，不過這也喪失了像我們這種把打牌當休閒娛樂的人玩牌的宗旨，快樂享受才是最重要的，所以老話一句，我忘記怎麼講了，反正就是，AI不是不好，看你如何使用以及自己的需求
